data/newsroom.toml

```diff
@@ -23,22 +23,341 @@ summary = "Company funded by Madrona Venture Group and Tola Capital to help comp
+img = "/logos/press/bizjournals.png"
+url = "https://www.bizjournals.com/seattle/news/2018/10/22/pulumi-fundraising-madrona-microsoft-tola-capital.html"
+
+[[coverage]]
+date = "22nd October 2018"
+title = "Pulumi Code Dev Platform Adds Premium Team Tier, Scores $15M Series A"
+img = "/logos/press/sdxcentral.png"
+url = "https://www.sdxcentral.com/articles/news/pulumi-code-dev-platform-adds-premium-team-tier-scores-15m-series-a/2018/10/"
+
+[[coverage]]
+date = "22nd October 2018"
+title = "Pulumi raises $15M in Series A funding, launches its SaaS multicloud app deployment service"
+img = "/logos/press/geekwire.png"
+url = "https://www.geekwire.com/2018/pulumi-raises-15m-series-funding-round-launches-multicloud-app-deployment-service/"
+
+[[coverage]]
+date = "22nd October 2018"
+title = "Pulumi raises $15M for its unique brand of infrastructure automation"
+img = "/logos/press/geekwire.png"
+url = "https://siliconangle.com/2018/10/22/pulumi-raises-another-15m-unique-brand-infrastructure-automation/"
 
 [[coverage]]
 date = "22nd October 2018"
 title = "Pulumi Launches Team Edition of Infrastructure as Code Platform"
-img = "/logos/press/eweek.jpg"
-url = "http://www.eweek.com/cloud/pulumi-launches-team-edition-of-infrastructure-as-code-platform/"
+img = "/logos/press/eweek.png"
+url = "http://www.eweek.com/cloud/pulumi-launches-team-edition-of-infrastructure-as-code-platform"
+
+[[coverage]]
+date = "22nd October 2018"
+title = "Pulumi Raises $15M For Its Infrastructure As Code Platform"
+img = "/logos/press/geekwire.png"
+url = "https://techcrunch.com/2018/10/22/pulumi-raises-15m-for-its-infrastructure-as-code-platform/"
+
+[[coverage]]
+date = "22nd October 2018"
+title = "Pulumi Raises $15M For Its Infrastructure As Code Platform"
+img = "/logos/press/geekwire.png"
+url = "https://www.startupworld.com/news/pulumi-raises-15m-for-its-infrastructure-as-code-platform/"
 
 [[coverage]]
 date = "22nd October 2018"
+title = "Pulumi snags $15 mln Series A"
+img = "/logos/press/geekwire.png"
+url = "https://www.pehub.com/2018/10/pulumi-snags-15-mln-series-a/"
+
+[[coverage]]
+date = "18th October 2018"
+title = "For developers, GitHub Actions takes automation a step forward"
+img = "/logos/press/techtarget.png"
+url = "https://searchsoftwarequality.techtarget.com/news/252450962/For-developers-GitHub-Actions-takes-automation-a-step-forward"
+
+[[coverage]]
+date = "15th October 2018"
+title = "GitHub introduces GitHub Actions to further automate the art of software development"
+img = "/logos/press/geekwire.png"
+url = "https://www.geekwire.com/2018/github-introduces-github-actions-automate-art-software-development/"
+
+[[coverage]]
+date = "15th October 2018"
 title = "Pulumi Bridges The Gap Between Cloud-Native Development And Infrastructure As Code"
 img = "/logos/press/forbes.png"
-url = "https://www.forbes.com/sites/janakirammsv/2018/10/16/pulumi-bridges-the-gap-between-cloud-native-development-and-infrastructure-as-code/"
```

Comment:
intentional?

Reply:
I think it's some kind of tracking info. When you take it out, a new one gets generated.